examples/examples.php

```diff
@@ -0,0 +1,45 @@
+<?php
```

Comment:
Do we need this file at all? This is the one that just prints a load of stuff out, right?